docs/integrations/microsoft-azure/azure-cache-for-redis.md

```diff
@@ -26,28 +26,200 @@ Azure service sends monitoring data to Azure Monitor, which can then [stream dat
 
 You must explicitly enable diagnostic settings for each Azure Cache for Redis you want to monitor. You can forward logs to the same event hub provided they satisfy the limitations and permissions as described [here](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/diagnostic-settings?tabs=portal#destination-limitations).
 
-When you configure the event hubs source or HTTP source, plan your source category to ease the querying process. A hierarchical approach allows you to make use of wildcards. For example: `Azure/RedisCache/Logs`, `Azure/RedisCache/Metrics`.
```

Comment:
Need to add Activity Logs section in log and metric types refer https://help.sumologic.com/docs/integrations/microsoft-azure/azure-storage/#log-and-metric-types